src/components/Plugin/SafeSnap/Config.vue

```diff
@@ -1,149 +1,69 @@
 <template>
-  <form @submit.prevent="handleSubmit">
-    <div class="mb-2 text-center">
-      <h4 class="mb-3">Transactions</h4>
-      <div v-if="adding">
-        <UiButton class="width-full mb-2">
-          <input
-            v-model="newEntry.to"
-            class="input width-full text-center"
-            placeholder="Target address"
-            required
-          />
-        </UiButton>
-        <UiButton class="width-full mb-2">
-          <input
-            v-model="newEntry.value"
-            class="input width-full text-center"
-            placeholder="Value"
-            required
-          />
-        </UiButton>
-        <UiButton class="width-full mb-2">
-          <input
-            v-model="newEntry.data"
-            class="input width-full text-center"
-            placeholder="Data"
-            required
-          />
-        </UiButton>
-        <UiButton class="width-full mb-2">
-          <select
-            v-model="newEntry.operation"
-            class="input width-full text-center"
-            required
-          >
-            <option v-bind:value="'0'" selected="selected">Call</option>
-            <option v-bind:value="'1'">Delegatecall</option>
-          </select>
-        </UiButton>
-        <UiButton @click="adding = false" class="mb-2">Back</UiButton>
-        <UiButton :disabled="!isValid" @click="addTx" class="button--submit">
-          Add
-        </UiButton>
-      </div>
-      <div v-else>
-        <div
-          v-for="(tx, i) in input.txs"
-          :key="i"
-          class="mb-3 p-4 border rounded-2 text-white text-center"
-        >
-          <PluginSafeSnapTransactionPreview :transaction="tx" />
-          <UiButton v-if="input" @click="removeTx(i)" class="width-full mb-2">
-            Remove
-          </UiButton>
-        </div>
-        <UiButton @click="adding = true" class="width-full mb-2">
-          Add
-        </UiButton>
-        <UiButton @click="handleSubmit" class="button--submit width-full">
-          Confirm
-        </UiButton>
+  <Block title="SafeSnap Plugin">
```

Comment:
Can we change this title to `[Transactions: {safe address}]({link to safe})`?
e.g. **Safe Transactions:** [0xFcC4...8680](https://gnosis-safe.io/app/#/safes/0xFcC41c4614bD464bA28ad96f93aAdaA7bA6c8680/transactions)